Documentation/NuttxUserGuide.html

```diff
@@ -10139,7 +10139,6 @@ <h3><a name="shmdt">2.12.4 <code>shmdt</code></a></h3>
 <li>struct tcb_s
 <li>mqd_t
 <li>sem_t
-<li>WDOG_ID
 <li>pthread_key_t
```

Comment:
Should this be replaced with struct wdog_s versus just removing WDOG_ID?

NOTE:  The wdog interfaces derive from the VxWorks wdog interfaces which does use the type WDOG_ID.  So this introduces another deviation from the interface definitions.  VxWorks is not a standard we have to follow, however, so that is not necessarily a problem but we probably should remove the statement that these interface functions derive from VxWorks.

Reply:
I will remove WDOG_ID and VxWork in the new patch.